frontend/src/scenes/retention/RetentionTable.tsx

```diff
@@ -88,14 +88,14 @@ export function RetentionTable({
                 size="small"
                 className="retention-table"
                 pagination={{ pageSize: 99999, hideOnSinglePage: true }}
-                rowClassName={'cursor-pointer'}
+                rowClassName={!dashboardItemId ? 'cursor-pointer' : ''}
```

Comment:
super nit but just for readability, this logic can be flipped so you don't need negation

Reply:
Yeah it's best practice to not have a negation on a boolean but it felt weird to me to have the `' '` as the "default" condition